public/js/ConversationController.js

```diff
@@ -46,4 +46,16 @@
         var id = setInterval(updateConversation, 1000);
 
     }]);
+
+    app.directive("chatAppConversation", function() {
```

Comment:
Doing a quick scan online, it looks more normal for a directive to be in its own file, or in a file with a series of directives or in a file with all directives.

Its more so that its easier to know where to look for them when looking at file and folder names.